### ISSUE TITLE
"OK" and "Cancel" buttons present in the dialog is not adopting the Text Spacing

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/fluentui-blazor#3692]
**Githubs Tags**:  #A11ySev2; #DesktopWeb; #A11yMAS; #A11yTCS; #Win11; #WCAG1.4.12; #FTP; #ChromiumEdge;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility after verification of the bug".**

## Environment Details:
Application Name: Fluent UI Blazor
Page URL: https://www.fluentui-blazor.net/Dialog
Chromium Edge Version 135.0.3179.73 (Official build) (64-bit)

## Pre-Requisites:

Install Stylus Extension: [Stylus (google.com)](https://chromewebstore.google.com/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne)
Include below lines of code and save it.
```
* {line-height: 1.5 !important;
letter-spacing: 0.12em !important; 
word-spacing: 0.16em !important;
}
p {margin-bottom: 2em !important;}
```

## Repro Steps:

1. Launch #URL: https://www.fluentui-blazor.net/Dialog
2. Tab to "Open Dialog" present in "Customized Dialog" and press enter to open.
3. Apply Text Spacing and Refresh.
4. Observe "Save" and "Cancel" button are not adapting to Text Spacing.

## Actual Result:

"OK" and "Cancel" buttons present in the dialog is not adopting the Text Spacing.

Similar issue is observed in below scenario:

- Placeholder text in text input field: https://www.fluentui-blazor.net/TextArea

## Expected Results:

"OK" and "Cancel" buttons present in the dialog must adapt to the Text Spacing.

## User Impact:

Increased text spacing helps users with dyslexia by reducing visual crowding and improving readability. Failure to adopt text spacing can make it difficult for these users to navigate and use the dialog effectively.

## WCAG Reference:

https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html

## Attachments:

![Image](https://github.com/user-attachments/assets/fee76a8d-2141-4dd3-ad40-722c0003d380)

**Comment by vnbaaij** [comment]
Hi @ManishaC1. Are you from Microsoft? If so, please reach out internally.

**Comment by MarvinKlein1508** [comment]
This doesn't work for `FluentButton` in general. I try to fix it :)
